app/lib/package/backend.dart

```diff
@@ -641,12 +643,14 @@ class PackageBackend {
 
   // Uploaders support.
 
-  Future<api.SuccessMessage> addUploader(
-      String packageName, String uploaderEmail) async {
-    uploaderEmail = uploaderEmail.toLowerCase();
+  Future<account_api.InviteStatus> inviteUploader(
+      String packageName, api.InviteUploaderRequest invite) async {
+    InvalidInputException.checkNotNull(invite?.email, 'email');
```

Comment:
Calling without `invite` is that not an `ArgumentError.checkNotNull(invite, 'invite');`